packages/hardhat-core/src/internal/hardhat-network/provider/modules/logger.ts

```diff
@@ -1,12 +1,18 @@
 import util from "util";
 
 export class ModulesLogger {
-  public enabled = false;
+  private _enabled = false;
   private _logs: Array<string | [string, string]> = [];
   private _titleLength = 0;
+  private _indentEnabled = false;
+  private _indent = 4;
```

Comment:
If it's never changed it should be a constant or readonly field.
```suggestion
  private readonly _indent = 4;
```